### PR TITLE
Release 1.5.0

### PR DESCRIPTION
**Pull request by jpogran**
- Update required dependencies and automatic tests
- Release 1.5.0

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 1.5.0 (2022-02-07)
+
+ENHANCEMENT:
+
+- Implement Proxy usage ([#11](https://github.com/hashicorp/js-releases/pull/11))
+
+DEPENDENCIES:
+
+- Upgrade openpgpjs to v5.0.1 stable ([#12](https://github.com/hashicorp/js-releases/pull/12))
```
Comment (dbanck) [inline CHANGELOG:9]: We upgraded openpgjs to 5.1.0 in https://github.com/hashicorp/js-releases/pull/15

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# 1.5.0 (2022-02-07)
```
Comment (dbanck) [inline CHANGELOG:1]: What do you think about renaming this file to `CHANGELOG.md`?

**Review by dbanck — APPROVED** [review]
LGTM! Just two minor things

File: CHANGELOG (hunk 1 above)
Comment (jpogran) [reply CHANGELOG]: Added two lines since it was technically a code change to upgrade to v5.0.1, whereas 5.1.0 was just a bump

**Review by dbanck — APPROVED** [review]

**Pull request merged**